pegasus/sites.v3/code.org/views/ai_vid_whyaimatters.erb

```diff
@@ -0,0 +1,3 @@
+<div>
+  <%=view :display_video_thumbnail, id: "why_ai_matters", video_code: "dWRnCXbUDgA", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-why-ai-matters.mp4" %>
```

Comment:
The `display_video_thumbnail` view isn't tab-navigable, i.e. I can't hit "play" using a keyboard. What I don't know is the best way to fix that. Maybe in the display_video_thumbnail partial, this img tag
```
%img.play{:src => "/shared/images/play-button.png", :alt => "play button"}
```
needs to be wrapped in an anchor or `button` tag? https://stackoverflow.com/questions/23475534/make-img-tag-tabbable-respond-to-tab-key-navigation

Looks like YouTube uses an `a` tag, but that goes to another page ...
<img width="473" alt="image" src="https://user-images.githubusercontent.com/9142121/232605583-99649b09-1f8d-4ebe-a784-aac99e048418.png">

Reply:
The `display_video_thumbnail` partial is used all over the site — I think a fix like this would be better in it's own issue/PR so we can adequately test it out. I'll make a note of this!

Reply:
Created a ticket here: https://codedotorg.atlassian.net/browse/ACQ-533?atlOrigin=eyJpIjoiOWRhZGU3MzgyMTAxNDgzMmJmMzk0NGU3M2QwNmJiZDMiLCJwIjoiaiJ9

Reply:
Great, thanks!